### ISSUE TITLE
Frame color in mainwindow example

### ISSUE THREAD

**Issue by JulienMaille** [wangwenx190/framelesshelper#87]
https://github.com/wangwenx190/framelesshelper/blob/0bfefb25d0c28ec792d48e0b9e6c969361d8683b/examples/mainwindow/mainwindow.cpp#L139

Why don't you use the frame color here?
`painter.setPen({Utilities::getNativeWindowFrameColor(isActiveWindow()), 1});`

**Comment by wangwenx190** [comment]
Yes you are right. There are some other small issues that need to be fixed, I'll push new code tomorrow to fix them all.

**Comment by JulienMaille** [comment]
@wangwenx190 despite the latest commits, the border is always grey

**Comment by wangwenx190** [comment]
OK. Let me check. First glance of the code doesn't find anything wrong.

**Comment by JulienMaille** [comment]
I agree, the code looks fine. Please note this is tested on win11, I can't test win10 right now.

**Comment by wangwenx190** [comment]
I can test Win10 and Win7 tomorrow. Currently I'm using a Win11 PC as well.

**Comment by wangwenx190** [comment]
Tested on both Win11 and Win10. The border color is normal on Win10, but on Win11 it is always gray indeed. However, I tested multiple Qt-based applications on Win11 and found that it seems it's a common issue of Qt-based applications, maybe it's a bug of Qt itself, currently still not sure, need further verify and investigate.

**Comment by JulienMaille** [comment]
It seems the gray border we see on Win10 is not the one drawn by framelesshelper.

**Comment by wangwenx190** [comment]
Yes, looks like it's the standard window frame provided by the OS. And I found that some things have changed on Windows 11. Previously I was able to get rid of the whole window frame, but now on Win11 it's no longer possible. No matter what I do, Windows will always add a thin frame border to the window, that's what we observed from the MainWindow example, the gray border. But there's one thing I still don't understand, despite of the window frame, we also draw our own frame border in the paint event, but on win11 it can't be seen anywhere. Maybe it's covered by something? Not sure yet.

**Comment by JulienMaille** [comment]
Could have something to do with the way they implemented round corners?
Despite the grey color, it works quite fine on win11, I've not noticed any bug yet.

**Comment by wangwenx190** [comment]
> Could have something to do with the way they implemented round corners?
>

Not sure, I tested this just now, I use DWM api to change the window frame corner to square, the issue still exist.

**Comment by wangwenx190** [comment]
Use the following code if you want to try:
```cpp
const DWM_WINDOW_CORNER_PREFERENCE dwcp = DWMWCP_DONOTROUND;
DwmSetWindowAttribute(hwnd, DWMWA_WINDOW_CORNER_PREFERENCE, &dwcp, sizeof(dwcp));
```
Note that this is only supported on Win11, use this enum on Win10 will fail.
